### ISSUE TITLE
\o not working for some cases.

### ISSUE THREAD

**Issue by skumagai** [jcfaria/Vim-R-plugin#192]
I notice \o sometimes doesn't work.

 `a  <- 10` works, but `a = 10` doesn't. The latter gets the following error:
```
Error in print.default(a = 0) : argument "x" is missing, with no default
```
in R's REPL. Additionally
  
```R
d <- data.frame(value=rnorm(10), cat=sample(letters[1:2], 10, replace=TRUE)) 
```
works, but
```R
d <- data.frame(value=rnorm(10),
                cat=sample(letters[1:2], 10, replace=TRUE)).
```
doesn't. I type \o after selecting these two lines  in the visual mode. Vim briefly shows INVALID at the bottom of the window.

I guess \o not working with selected multiple lines is not surprising given its explanation: Line (evaluate and insert the output as comment). However, similarly explained commands \l (Line) and \d work with selection. It would be great if \o can also work with selection.

**Comment by jalvesaq** [comment]
When you runs a command over a range of lines, Vim repeatedly executes the command for each line. When incomplete commands are sent to R Console, R waits for the next line before executing the command. The line sent by `<LocalLeader>o` is not sent to R Console. It is evaluated immediately by the *vimcom* library (C code).

The feature was added to the Vim-R-plugin with the intention of allowing to put in the source code the output of something that would be printed in R Console (for example, when you were sending the code to someone who does not use R). So, it was thought to be used over the last line of a code like this:
```r
d <- data.frame(value=rnorm(10),
                cat=sample(letters[1:2], 10, replace=TRUE))
d
```
When you were sending multiple lines what the plugin really was trying to do was:
```r
print(d <- data.frame(value=rnorm(10),)
print(                cat=sample(letters[1:2], 10, replace=TRUE))'
```
Of course, `print(a = 10)` does not work either.
You can see what is being sent to *vimcom* if you put in your `~/.Rprofile`:
```r
options(vimcom.verbose = 2)
```
I added a paragraph to the documentation explaining how `<LoalLeader>o` works. I also added a key binding for `<LocalLeader>o` in Visual mode warning that it does not work with a selection of lines.

**Comment by skumagai** [comment]
Thank you for the explanation of the inner workings of \o and vimcom. Now, I understand why what I did is not sensible.

It never occurred to me to store the results in variables..., because my main use case of \o was to embed throw-away results (e.g., output of `summery` not used anywhere else). Then, when I started to use \o with multiline expressions involving `dplyr`, I got into this situation.

Again thank you for the explanation. I'll adjust my workflow.
